content/en/blog/_posts/2021-04-22-gateway-api/2021-04-22-gateway-api.md

```diff
@@ -0,0 +1,199 @@
+- **Flexible conformance** - This is where the Gateway API aims to improve the portability Ingress through more [flexible conformance](https://gateway-api.sigs.k8s.io/concepts/guidelines/#conformance). With core (mandatory support), extended (portable if supported), and custom (no portability guarantee) conformance levels, the API is explicit about its portability without constraining implementations-specific capabilities.
+
+
+### What does the Gateway API look like?
+
+The Gateway API introduces a few new resource types:
+
+- **GatewayClasses** are a cluster-scoped resource that act as templates to explicitly define behavior for Gateways derived from them. This is similar in concept to StorageClasses, but for Gateways, our Ingress-like object.
+- **Gateways** are the deployed instances of GatewayClasses. They are the logical representation of the data-plane which performs routing, which may be in-cluster proxies, hardware LBs, or cloud LBs.
+- **HTTPRoutes** are the matching, filtering, and routing rules that get applied to Gateways, which results in the configuration of the underlying data-plane. There are also other kinds of Routes that exist too (since they are protocol-specific).
+- **xRoutes** are not a resource itself, but represent that Routes are protocol-specific. This provides a path for future evolution.  TLSRoutes, TCPRoutes, and UDPRoutes already have proposals with the future possibility of other protocols such as GRPCRoutes.
+
+![gateway-api-resources](./gateway-api-resources.png)
+
+### Gateway Controller Implementations
+
+The good news is that although Gateway is in [v1Alpha1](https://github.com/kubernetes-sigs/gateway-api/releases), there are already several [Gateway controller implementations](https://gateway-api.sigs.k8s.io/references/implementations/) that you can run. Since it’s a standardized spec, the following example could be run on any of them and should function the exact same way. Check out [getting started](https://gateway-api.sigs.k8s.io/guides/getting-started/) to see how to install and use one of these Gateway controllers.
+
+## Getting Hands-on with the Gateway API
+
+In the following example, we’ll demonstrate the relationships between the different API Resources and walk you through a common use case:
+
+*   Team foo has their app deployed in the foo Namespace. They need to control the routing logic for the different pages of their app.
+*   Team bar is running in the bar Namespace. They want to be able to do blue-green rollouts of their application to reduce risk.
+*   The platform team is responsible for managing the load balancer and network security of all the apps in the Kubernetes cluster.
+
+The following foo-route does path matching to various Services in the foo Namespace and also has a default route to a 404 server. This exposes foo-auth and foo-home Services via `foo.example.com/login` and `foo.example.com/home` respectively.:
+
+
+```
```

Comment:
If you append `yaml` to this line, the website adds syntax highlighting.

Reply:
+1
Reading without highlighting is very taxing.

@sftim is there a way to make some parts of YAML bold? Like in the Administrative Delegation section, we want to focus on the labels in the HTTPRoute resource.

Reply:
The syntax highlighter doesn't offer you that. What you can add is explanatory YAML comments.

Reply:
@hbagdi lmk if there are any portions that we should add comments to